frontend/src/Components/NotificationsPopup.tsx

```diff
@@ -76,50 +79,80 @@ export default function NotificationsPopup(props: NotificationsPopupProps) {
             });
     }, [fetchNotifications, subscribe]);
 
-    const handleNotificationClick = (e: React.MouseEvent<HTMLAnchorElement>, notify: NotificationInfo) => {
+    const handleNotificationClick = (notify: NotificationInfo) => (e: React.MouseEvent<HTMLAnchorElement>) => {
         api.notifications.read(notify.id)
             .then()
             .catch();
```

Comment:
Output error through `console.error` maybe? Silently swallowing error will make it really hard to debug.